test/cmd/certs.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+os::test::junit::declare_suite_start "cmd/admin/certs-validation"
+
+CERT_DIR="${BASETMPDIR:-/tmp}/certs"
+mkdir -p "${CERT_DIR}"
+
+# oadm ca create-signer-cert should generate certificate for 5 years by default
+os::cmd::expect_success_and_not_text \
```

Comment:
We support multi-line commands, so you can do something like the following if you think it reads better:

```sh
os::cmd::expect_success_and_not_text "oadm ca create-signer-cert --cert=${CERT_DIR}/ca.crt          \
                                                                 --key=${CERT_DIR}/ca.key           \
                                                                 --serial=${CERT_DIR}/ca.serial.txt \
                                                                 --overwrite=true" \
                                     'WARNING: .* is greater than 5 years'
```